ls1intum/Artemis
# [Bugfix] Displaying columns for small screens in overviews

### Checklist
- [x] I tested *all* changes and *all* related features (admin) on the test server https://artemistest.ase.in.tum.de.

### Problem
The ID Title disappears when the exam overview/student exam page is displayed on small screens.
![IdNotDisplayedProperly](https://user-images.githubusercontent.com/63976129/129282636-66636080-0d21-4ac9-b2e7-4c5abe6b3046.gif)

### Description

Always display the ID title by removing the `d-none` class.

I did not understand the reasoning behind adding the `d-none` class, feel free to add a comment in case I might miss why this is valuable.
My knowledge is limited to using the display property [hide for increased accessibility ](https://www.nomensa.com/blog/how-improve-web-accessibility-hiding-elements)- I am not aware of further usages which might apply here? 

### Steps for Testing


1. Create an Exam and participate with one student
2. Navigate to the student exam view (_Student Exams_ | _View_)
3. Shrink the screen and make sure that the ID title is always displayed
4. Repeat step 3 for the _Exam Management_ overview
5. Repeat step 3 for the _Exercise Overview_ (programming exercises and modeling exercises were modified here - you might notice another problem here, an issue for that does already exist https://github.com/ls1intum/Artemis/issues/3993 and respective PR https://github.com/ls1intum/Artemis/pull/3909)
6. Repeat step 3 for the _Exam Exercise Group Overview_

### Screenshots


Fixed version:
![IdNotDisplayedProperlyFixed](https://user-images.githubusercontent.com/63976129/129282857-42d612b2-950a-4418-9d36-dd577618ba08.gif)


### Review Progress
- Code Review
- Manual Tests
